bin/pygrb/pycbc_make_offline_grb_workflow

```diff
@@ -411,7 +411,7 @@ if post_proc_method == "PYGRB_OFFLINE":
     # The content and structure of pp_files are described in the definition
     # of _workflow.setup_pygrb_pp_workflow
     pp_files = _workflow.setup_pygrb_pp_workflow(wflow, pp_dir, seg_dir,
-                                      sciSegs[ifo][0], inspiral_files,
+                                      sciSegs[ifo][0], bank_files[0], inspiral_files,
```

Comment:
Is the first item of `bank_files` always guaranteed to be the full bank? Or is `bank_files` always going to contain a single item? This is not obvious from the code. A comment would be useful, or maybe a more self-explanatory choice of variables? Do not bother if too complicated though.

Reply:
Good point. If we were to run with two bank files this would become an issue. If there's only one bank then the `bank_files` object should just the single file. I suppose it would be worth having the workflow generator allow for multiple bank files. I don't think it would be complicated to do that, but that case would cause problems with the template hash/ID mapping.

Reply:
I do not see what use case we might want to support where there are two banks in a single workflow. Not in the immediate future, at least.

Reply:
I would do the "Multiple banks not supported" check early on, immediately after the definition of `bank_files` in `pycbc_make_offline_grb_workflow`. Then define `full_bank_file = bank_files[0]`. In that way you make it obvious from the code that there is a single full bank file.